src/plugins/content_management/public/components/card_container/card_embeddable.tsx

```diff
@@ -5,13 +5,15 @@
 
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { EuiCard, EuiCardProps } from '@elastic/eui';
+import { EuiCard, EuiCardProps, EuiToolTip } from '@elastic/eui';
 
 import { Embeddable, EmbeddableInput, IContainer } from '../../../../embeddable/public';
 
 export const CARD_EMBEDDABLE = 'card_embeddable';
 export type CardEmbeddableInput = EmbeddableInput & {
   description: string;
+  toolTipContent?: string;
```

Comment:
Nit: Should be EuiToolTipProps["toolTipContent"]

Reply:
The EuiToolTipProps do not have the toolTipContent attribute. I just need to pass the string, and use it as the tooltips of description.